File: sample-conference-videochat/sample-conference-videochat/Components/Calls/StreamTitleView/StreamTitleView.m
//
//  StreamTitleView.m
//  sample-conference-videochat
//
//  Created by Injoit on 15.06.2020.
//  Copyright © 2020 Quickblox. All rights reserved.
//

#import "StreamTitleView.h"

@implementation StreamTitleView

- (void)setupStreamTitleViewOnLive:(BOOL)onLive {
    self.frame = CGRectMake(0, 0, 72.0f, 20.0f);
    self.image = onLive == YES ? [UIImage imageNamed:@"live_streaming"] : [UIImage imageNamed:@"end_stream"];
}

@end

File: sample-conference-videochat-swift/sample-conference-videochat-swift/Components/Calls/StreamTitleView/StreamTitleView.swift
//
//  StreamTitleView.swift
//  sample-conference-videochat-swift
//
//  Created by Injoit on 5/22/20.
//  Copyright © 2020 QuickBlox. All rights reserved.
//

import UIKit

class StreamTitleView: UIImageView {
    
    func setupStreamTitleViewOnLive(_ onLive: Bool) {
        self.frame = CGRect(x: 0, y: 0, width: 72.0, height: 20.0)
        image = onLive == true ? #imageLiteral(resourceName: "live_streaming") : #imageLiteral(resourceName: "end_stream")
    }
}
